crates/uv/src/commands/tool/install.rs

```diff
@@ -56,8 +56,12 @@ pub(crate) async fn install(
         .connectivity(connectivity)
         .native_tls(native_tls);
 
+    let python_request = python.as_deref().map(ToolchainRequest::parse);
+
+    // Pre-emptively identify a Python interpreter. We need an interpreter to resolve any unnamed
+    // requirements, even if we end up using a different interpreter for the tool install itself.
```

Comment:
This is somewhat annoying.

Reply:
(Not a behavior change, just forced to explain why now. I'm not sure how to avoid it. I can try to make it lazy.)